apps/server/test/lexical/server/code_intelligence/completion/translations/macro_test.exs

```diff
@@ -612,4 +612,50 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Translations.MacroTest do
       refute boosted?(completion)
     end
   end
+
+  test "test has three completions", %{project: project} do
+    assert {:ok, completions} =
+             project
+             |> complete(inside_exunit_context("test|"))
+             |> fetch_completion("test ")
+
+    assert [%Completion.Item{}, %Completion.Item{}, %Completion.Item{}] = completions
+  end
+
+  test "test completion snippets", %{project: project} do
+    assert {:ok, completions} =
+             project
+             |> complete(inside_exunit_context("test|"))
+             |> fetch_completion("test ")
+
+    assert %Completion.Item{
```

Comment:
Why not just match out the completions above rather than use `get_in` and `Access.at(..)`?

```elixir
assert {:ok, [stub, with_body, with_context]} = ...
```

Reply:
I just got a little used to tossing around the lensing functions (e.g. `get_in`). But I like your idea better.